live-examples/js-examples/string/meta.json

```diff
@@ -1,5 +1,47 @@
 {
     "pages": {
+        "stringSubstr": {
+            "baseTmpl": "tmpl/live-js-tmpl.html",
+            "exampleCode": "live-examples/js-examples/string/string-substr.html",
+            "fileName": "string-substr.html",
+            "title": "JavaScript Demo: String.substr()",
+            "type": "js"
+        },
+        "stringConcat": {
+            "baseTmpl": "tmpl/live-js-tmpl.html",
+            "exampleCode": "live-examples/js-examples/string/string-concat.html",
+            "fileName": "string-concat.html",
+            "title": "JavaScript Demo: String.concat()",
+            "type": "js"
+        },
+        "stringEndsWith": {
+            "baseTmpl": "tmpl/live-js-tmpl.html",
+            "exampleCode": "live-examples/js-examples/string/string-endswith.html",
+            "fileName": "string-endswith.html",
+            "title": "JavaScript Demo: String.endsWith()",
+            "type": "js"
+        },
+        "stringPadEnd": {
+            "baseTmpl": "tmpl/live-js-tmpl.html",
+            "exampleCode": "live-examples/js-examples/string/string-padend.html",
+            "fileName": "string-padend.html",
+            "title": "JavaScript Demo: String.padEnd()",
+            "type": "js"
+        },
+        "stringPadStart": {
+            "baseTmpl": "tmpl/live-js-tmpl.html",
+            "exampleCode": "live-examples/js-examples/string/string-padstart.html",
+            "fileName": "string-padstart.html",
+            "title": "JavaScript Demo: String.padStart()",
+            "type": "js"
+        },
+        "stringStartsWith": {
+            "baseTmpl": "tmpl/live-js-tmpl.html",
+            "exampleCode": "live-examples/js-examples/string/string-startswith.html",
+            "fileName": "string-startswith.html",
+            "title": "JavaScript Demo: String.startsWith()",
+            "type": "js"
+        },
```

Comment:
I'm not sure why this has these additional entries in meta.json, is it intentional?

Reply:
I think I was trying to avoid the conflict by adding Estelle's entries.